packages/ios-client/Sources/FishjamClient/webrtc/PeerConnectionManager.swift

```diff
@@ -201,15 +202,9 @@ internal class PeerConnectionManager: NSObject, RTCPeerConnectionDelegate {
         peerConnection.enforceSendOnlyDirection()
     }
 
-    private func drainCandidates() {
-        for candidate in queuedRemoteCandidates {
-            connection?.add(candidate, completionHandler: { _ in })
-        }
-    }
-
     /// Parses a list of turn servers and sets them up as `iceServers` that can be used for `RTCPeerConnection` ceration.
     private func setTurnServers(_ turnServers: [OfferDataEvent.TurnServer]) {
-        config?.iceTransportPolicy = .relay
+        isExWebrtc = turnServers.count == 0
```

Comment:
Does it need to be class level variable? It looks like it can be defined locally in `getSdpOffer`, as it used only there 🤔

Reply:
Good point